Added the following substitution

BB2A-Anthony opened the pull request:
Added the following substitution from line 6025.

https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/core/lib/functions.lib.php#L6025

# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "Fix", "Close" or "New" section*
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed textswith meaningful informations*


# Fix #[*issue_number Short description*]
[*Long description*]


# Close #[*issue_number Short description*]
[*Long description*]


# New [*Short description*]
[*Long description*]

eldy commented:
Something seems wrong with the PR. 5007 files were modified. May be you pushed the PR on the wrong branch...

eldy committed:
Merge pull request #12029 from altatof/limitlang

NEW : add MAIN_LANGUAGES_ALLOWED constant to limit languages displayed

eldy committed:
Merge pull request #12012 from frederic34/ficheinter_rec

restore unfinished old commit

eldy committed:
Merge pull request #12010 from laudeco/feature/12006-country-on-pdf-footer

[#12006] - adds the country in the address of the PDF's footer

eldy committed:
Merge pull request #11986 from atm-john/develop_dropdown_bookmark

NEW :  dropdown bookmark and search

eldy committed:
Fix bad br

eldy committed:
css

frederic34 committed:
duration of fichinter rec

aspangaro committed:
Fix Bank Fa-icon

aspangaro committed:
Remove constant MAIN_DISABLE_FONT_AWESOME_5

aspangaro committed:
Add bookmark fa-icon

Commit:
Fix project libStatus to use new dolGetStatus

Commit:
Fix dolGetStatus  param

aspangaro committed:
Add globe fa-icon

Commit:
Fix dolGetStatus  unused

Commit:
Fix supplier order libStatus to use new dolGetStatus

stickler-ci committed:
Fixing style errors.

Commit:
Fix Project LibStatus langs

Commit:
Merge branch 'develop_fix_missing_dolgetstatus' of github.com:atm-john/dolibarr into develop_fix_missing_dolgetstatus

stickler-ci committed:
Fixing style errors.

Ahuahuachi committed:
Add hidden options to send by email for all status

Ahuahuachi committed:
Merge remote-tracking branch 'Dolibarr/develop' into develop

lvessiller-opendsi committed:
NEW add country code in import product model

altairis-tof committed:
Merge branch 'develop' into prodstats

eldy committed:
Increase delay of hide of popup so we have a chance to click on content

Commit:
Merge branch 'develop' of github.com:Dolibarr/dolibarr into dev_contactdefault

stickler-ci committed:
Fixing style errors.

eldy committed:
Use small image instead of mini for preview on document list

eldy committed:
FIX access to public interface when origin email has an alias.

eldy committed:
NEW Add link to the public interface on the ticket card.

eldy committed:
Look and feel v11

eldy committed:
FIX escape email alias

eldy committed:
FIX Bad list of ticket on public interface for ticket emailcollector

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/compta/facture/card.php
	htdocs/core/lib/functions.lib.php
	htdocs/core/lib/product.lib.php
	htdocs/core/tpl/extrafields_list_search_input.tpl.php
	htdocs/fourn/facture/card.php
	htdocs/product/class/html.formproduct.class.php

frederic34 committed:
api supplier price delete

frederic34 committed:
api supplier get purchase prices

atm-maxime committed:
Add info about validators saying default is supervisor

atm-maxime committed:
Translations

atm-maxime committed:
Merge branch 'develop' of github.com:atm-maxime/dolibarr into new_validators

stickler-ci committed:
Fixing style errors.

frederic34 committed:
Update card.php

frederic34 committed:
lang file for holidays is holiday

eldy committed:
Fix js syntax error

eldy committed:
Fix syntax error

eldy committed:
NEW Code for extrafields uses the new array $extrafields->attributes

eldy committed:
FIX Look and feel v10

eldy committed:
FIX Look and feel v10

eldy committed:
Look and feel v11

eldy committed:
FIX Pb in units of shipments

eldy committed:
Fix migration

eldy committed:
Clean code of extrafields

eldy committed:
Fix syntax error

eldy committed:
Fix css

eldy committed:
FIX Load chart of account only if we click or select button "Load chart"

eldy committed:
FIX Load chart of account only if we click or select button "Load chart"

frederic34 committed:
Update contrat.class.php

eldy committed:
Merge pull request #12049 from frederic34/holidayz

lang file for holidays is holiday

eldy committed:
Merge pull request #12046 from frederic34/apisupplierproduct

api supplier price delete

eldy committed:
Merge pull request #12048 from frederic34/patch-17

Update card.php

eldy committed:
Merge pull request #12050 from frederic34/patch-18

Update contrat.class.php for doxygen

eldy committed:
Merge branch 'develop' into new-import-product-country-code

eldy committed:
Merge pull request #12039 from OPEN-DSI/new-import-product-country-code

NEW add country code in import product model

eldy committed:
Code comment

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/core/modules/import/import_csv.modules.php
	htdocs/core/modules/import/import_xlsx.modules.php
	htdocs/core/modules/modProduct.class.php
	htdocs/expedition/card.php

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge pull request #12034 from aspangaro/11_faicon

Fix Fa-icon

eldy committed:
Merge pull request #12032 from frederic34/ficheinter_rec

duration of fichinter rec

eldy committed:
Merge pull request #12035 from atm-john/develop_fix_missing_dolgetstatus

Fix : LibStatut missing dolgetstatus usage

eldy committed:
Merge branch 'develop' into patch-264

eldy committed:
Merge pull request #12015 from atm-maxime/new_validators

NEW expense request and holiday validator fields

ptibogxiv committed:
Rename traduction.php to type_translation.php

ptibogxiv committed:
Update type_translation.php

ptibogxiv committed:
Update member.lib.php

eldy committed:
Clean user card

eldy committed:
Trans

eldy committed:
Merge pull request #12026 from oscim/patch-6

Fix error in triggers process

eldy committed:
Merge pull request #12038 from Ahuahuachi/develop

NEW Add hidden options to send by email even for draft status

eldy committed:
Update dav.php

eldy committed:
Merge pull request #12020 from frederic34/patch-13

Update dav admin

eldy committed:
Merge pull request #11951 from aspangaro/11.0_accountancy_portal

NEW Accountancy - Add an option to hide FAQ on index and add specific widget in this area

eldy committed:
Debug use of widget on Accounting home page

eldy committed:
Update card.php

eldy committed:
Update interface_90_modSociete_ContactRoles.class.php

eldy committed:
Update interface_90_modSociete_ContactRoles.class.php

eldy committed:
Merge pull request #11829 from atm-florian/dev_contactdefault

NEW : Address/Contact by default on third parties

eldy committed:
Debug contact by default

eldy committed:
CSS

eldy committed:
Trans

eldy committed:
css

eldy committed:
FIX #12058

frederic34 committed:
use $this->db in box

eldy committed:
Update adherent.class.php

eldy committed:
Update card.php

NEW Add option MEMBER_CAN_CONVERT_CUSTOMERS_TO_MEMBERS

eldy committed:
Merge pull request #12059 from frederic34/boxfix

use $this->db in box

eldy committed:
Try to fix stickler

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

eldy committed:
css

eldy committed:
NEW Can edit the price of predefined product during adding in documents

frederic34 committed:
remove stock data from getsupplierprice

stock data can be retrieved with get

laudeco committed:
export some other fields on the expense reports

When the user requests the export of the expense reports, some fields
were missing such as the bank account information of the user.

This PR adds the support of all those fields on the expense report
export. Now when you export the expense report you have :
* The bank name
* The bank code
* The IBAN
* The BIC
* The owner
* The owner address

eldy committed:
Merge pull request #11920 from ptibogxiv/patch-264

NEW translation for member type

eldy committed:
Clean code to use the new $extrafields->attributes array

eldy committed:
Removed ccampbell library

eldy committed:
Update TCPDF to 6.3.2

eldy committed:
Fix lib version

eldy committed:
Fix versions/lib

eldy committed:
Restore original official files

eldy committed:
Fix missing dolibarr patch

eldy committed:
Patch

eldy committed:
Commit updated lock file

eldy committed:
Update Restler to 3.0RC6 (last bug fixes of branch v3)

eldy committed:
Update lib versions

eldy committed:
Doxygen

eldy committed:
css

eldy committed:
Merge pull request #12062 from frederic34/patch-13

remove stock data from getsupplierprice

eldy committed:
Merge pull request #11847 from ptibogxiv/patch-257

NEW: Option to allow to create members from third-party

eldy committed:
Merge pull request #11094 from altatof/prodstats

NEW Extend option ORDER_ADD_ORDERS_WITH_PARENT_PROD_IF_INCDEC for all virtual product stats (renamed into PRODUCT_STATS_WITH_PARENT_PROD_IF_INCDEC)

eldy committed:
Update ccountry.class.php

eldy committed:
Update ccountry.class.php

eldy committed:
Merge pull request #11371 from bafbes/abb90058

New:Show labels of countries in modulebuilder

eldy committed:
Update export.php

eldy committed:
Merge pull request #12063 from laudeco/feature/export_user_bank_account

NEW Add bank data of user into the expense report exports

stickler-ci committed:
Fixing style errors.

fuhraih committed:
FIX #11835

FIX #11835 edit Bank Statements in second Page and so on not working

eldy committed:
Trans

fuhraih committed:
Merge pull request #1 from fuhraih/fuhraih-patch-1

FIX #11835

eldy committed:
Trans

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

fuhraih committed:
Update companies.lang

eldy committed:
Clean code

eldy committed:
Trans

eldy committed:
Update releve.php

fuhraih committed:
Update dict.php

eldy committed:
Update releve.php

eldy committed:
Merge pull request #12065 from fuhraih/fix1

FIX #11835

ptibogxiv committed:
Update llx_product.sql

ptibogxiv committed:
Update product.class.php

ptibogxiv committed:
Update product.class.php

ptibogxiv committed:
Update 10.0.0-11.0.0.sql

eldy committed:
Enable new top dashboard by default

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
css

eldy committed:
Fight against colspan

frederic34 committed:
fix travis

frederic34 committed:
Update list.php

eldy committed:
Merge pull request #12067 from frederic34/patch-13

fix travis

eldy committed:
Merge branch 'develop' into patch-255

eldy committed:
Update 10.0.0-11.0.0.sql

eldy committed:
Merge pull request #11831 from ptibogxiv/patch-255

NEW auto set closing date and user on invoice

eldy committed:
Picto of user umage larger

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
css

frederic34 committed:
fix travis

eldy committed:
Merge pull request #12068 from frederic34/patch-13

fix travis

eldy committed:
Merge pull request #12066 from fuhraih/feature1

NEW #4301

eldy committed:
Fix php7.4 error

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy commented:
4,830 files modified ? Something is wrong.
You should try to make a git pull from the original stream then try to make the pr again.

eldy committed:
Merge branch 'develop' into patch-187

eldy committed:
Merge pull request #11340 from ptibogxiv/patch-187

NEW: capacity value in product

eldy committed:
Add more phpunit

eldy committed:
Merge pull request #10960 from atm-john/New_look_and_feel_appli

NEW: look and feel appli

eldy committed:
Update logo name

eldy committed:
Fix import of web site
Fix sort on constant setup page lost

eldy committed:
Responsive

ptibogxiv committed:
Update card.php

aspangaro committed:
Update new LDCompta Export

eldy committed:
Use fontawesome for img_warning

eldy committed:
Use a font awesome icon to show "shippable" flag on orders.

eldy committed:
Fix missing br

aspangaro committed:
Merge remote-tracking branch 'upstream/develop' into 11.0_LDcompta

frederic34 committed:
Update list.php

frederic34 committed:
Update list.php

frederic34 committed:
globe fontawesome

frederic34 committed:
Update fournisseur.commande.class.php

frederic34 committed:
Update box_factures_imp.php

frederic34 committed:
Update functions.lib.php

eldy committed:
Add image for members

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/langs/en_US/margins.lang
	htdocs/user/class/user.class.php

eldy committed:
Exclude medias from fixdosfiles

eldy committed:
Fix dos2unix files

eldy committed:
css

aspangaro committed:
NEW : Accountancy - List by account - Modify sortfield, nowrap on amount and sum of sous-total

ptibogxiv committed:
Update api_setup.class.php

ptibogxiv committed:
Update api_setup.class.php

eldy committed:
NEW Add column of module source and POS terminal in invoice list.

eldy committed:
Fix trans

eldy committed:
Rename const

eldy committed:
Trans

eldy committed:
Fix link

ptibogxiv committed:
Update api_setup.class.php

eldy committed:
Fix sort lost after uploading a file

eldy committed:
CSS

frederic34 committed:
doxygen

frederic34 committed:
get relative path for favicon

eldy committed:
NEW Can filter on label on invoice in accounting vendor binding pages

eldy committed:
NEW Look and fill v11: Introduce CSS "trforbreak"

eldy committed:
CSS

eldy committed:
CSS

eldy committed:
Look and feel v11

frederic34 committed:
fix search by roles

grandoc committed:
fix some translations

Commit:
Fix logo background and hidden conf

Commit:
Fix css name

grandoc committed:
Update commondocgenerator.class.php

grandoc committed:
update code

eldy committed:
NEW Can set a squarred icon on your company setup

eldy committed:
css

eldy committed:
Fix avoid deletion of file if we still need it

eldy committed:
typo

eldy committed:
NEW Use the squarre logo as favicon of pages

eldy committed:
NEW Use the squarre logo as favicon of pages

eldy committed:
Update thumbs of themes

eldy committed:
NEW Look and feel v11: Some setup pages are by default on edit mode

eldy committed:
Fix option ADD_UNSPLASH_LOGIN_BACKGROUND

eldy committed:
NEW Look and feel v11: Some setup pages are by default on edit mode

eldy committed:
Align code of auguria menu handler with new architecture

Commit:
Merge branch 'develop' of github.com:Dolibarr/dolibarr into develop_fix_logo_bg

eldy committed:
Look and feel v11

eldy committed:
Try to fix #12097

eldy committed:
Merge pull request #12074 from aspangaro/11.0_LDcompta

Update new LDCompta export

eldy committed:
Merge pull request #12079 from frederic34/patch-17

Update box_factures_imp.php

eldy committed:
Fix phpunit of BOM

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '9.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/core/class/link.class.php

eldy committed:
Merge branch '10.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/core/class/link.class.php

eldy committed:
Merge pull request #12090 from frederic34/patch-19

doxygen

eldy committed:
Merge pull request #12071 from ptibogxiv/patch-266

NEW: support Net Measure in product's card.php

eldy committed:
Merge pull request #12094 from grandoc/new_branch_11_10_2019

fix some translations

eldy committed:
Merge pull request #12095 from atm-john/develop_fix_logo_bg

Fix logo container css name and add hidden conf to remove background

eldy committed:
Merge pull request #12078 from frederic34/patch-16

Update fournisseur.commande.class.php

eldy committed:
Merge branch 'develop' into patch-22

eldy committed:
Merge pull request #12092 from frederic34/patch-22

get relative path for favicon

eldy committed:
Update list.php

eldy committed:
Update list.php

eldy committed:
Merge branch 'develop' into patch-14

eldy committed:
Merge pull request #12076 from frederic34/patch-14

Update list.php

eldy committed:
Merge pull request #12077 from frederic34/patch-15

globe fontawesome

eldy committed:
Merge branch 'develop' into patch-18

eldy committed:
Merge pull request #12080 from frederic34/patch-18

fontawesome chart-bar for stats icon

eldy committed:
Update listbyaccount.php

eldy committed:
Merge pull request #12085 from aspangaro/11.0_accounting

NEW : Accountancy - List by account - Modify sortfield, nowrap on amo…

eldy committed:
Fix missing token in form
Fix date lost after search

eldy committed:
Merge pull request #12093 from frederic34/patch-23

fix search by roles

eldy committed:
Fix missing token

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix trans

eldy committed:
Merge pull request #12086 from ptibogxiv/patch-267

NEW contacts type dictionnary in api_setup.class.php

eldy committed:
Fix phpcs

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy commented:
PR is not valid. It include several thousand of commits from other. probably done on wrong branch.
